spec/spec_helper.rb

```diff
@@ -58,4 +58,7 @@
 
   config.before(:all) { ActiveRecord::Base.skip_callbacks = true }
   config.before(:each) { ActionMailer::Base.deliveries.clear }
+
+  config.filter_run focus: true
```

Comment:
Just allows you to set `focus: true` option on specs and it only runs those specs if you're running a filewatcher to re-run specs, for example. Not really relevant to this PR. I can take it out if you want.